modules/saml/lib/SP/LogoutStore.php

```diff
@@ -14,12 +14,19 @@ class sspmod_saml_SP_LogoutStore {
 	 */
 	private static function createLogoutTable(\SimpleSAML\Store\SQL $store) {
 
-		if ($store->getTableVersion('saml_LogoutStore') === 1) {
+		if ($store->getTableVersion('saml_LogoutStore') === 2) {
```

Comment:
Can we store the result of `getTableVersion()` in a variable and use it in the conditions instead of evaluating the function twice? Something like:

```php
$tableVer = $store->getTableVersion('saml_LogoutStore');
if ($tableVer === 2) {
    return;
} elseif ($tableVer === 1) {
```